### DISCUSSION TITLE
Autoincrement numeric ID

### DISCUSSION THREAD

**Discussion by FrancocDev** [payloadcms/payload#1366] [Q&A] [answered]
Hi, I'm trying to use numeric id's and use the defaultValue as the last post + 1. 
Thanks and sorry for my english.

**Comment by thgh** [comment, marked as answer]
[Here is some inspiration](https://github.com/payloadcms/payload/issues/1243#issuecomment-1308992274)

**Reply by FrancocDev** [reply]
Thanks a lot!
